README.md

```diff
@@ -49,7 +49,9 @@ Stampit uses [three different kinds of prototypal OO](https://vimeo.com/69255635
 [![NPM](https://nodei.co/npm/stampit.png)](https://www.npmjs.com/package/stampit)
 
 Browsers: [![CDNJS](https://img.shields.io/badge/cdnjs.com--green.svg)](https://cdnjs.com/libraries/stampit)
-[![UNPKG](https://img.shields.io/badge/unpkg.com--green.svg)](https://unpkg.com/stampit)
+[![UNPKG](https://img.shields.io/badge/unpkg.com--green.svg)](https://unpkg.com/stampit@latest/dist/)
```

Comment:
I don't think this is a good way to show everything as it may confuse people what to use actually. CDNJS is also providing UMD build only.

And there is no point in serving anything else but UMD bundle for a browser. Those other two are using `require` which would not work without bundler and in that case people can probably use NPM directly as well.